package.json

```diff
@@ -37,5 +37,8 @@
     "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.10.3",
     "webpack-merge": "^4.1.5"
+  },
+  "peerDependencies": {
+    "bitmovin-player": "^8.31.0"
```

Comment:
I think we don't need the `bitmovin-player` in both, `devDependencies` and `peerDependencies`, right? I think having it in `peerDependencies` makes sense, so I'd suggestion removing it from `devDependencies` if there are no good reasons to keep it there.

Reply:
`peerDependencies` are not installed when you run `npm install`. They are useful to tell the consumers what dependencies your package need to be able to run. It is needed to keep `bitmovin-player` in `devDependencies` to be able to import the types from that package.

Reply:
Yes, I will update it.